PyViCare/PyViCareDeviceConfig.py

```diff
@@ -84,10 +84,8 @@ def asAutoDetectDevice(self):
             (self.asPelletsBoiler, r"Vitoligno|Ecotronic|VBC550P", []),
             (self.asRadiatorActuator, r"E3_RadiatorActuator", ["type:radiator"]),
             (self.asRoomSensor, r"E3_RoomSensor", ["type:climateSensor"]),
-            (self.asElectricalEnergySystem, r"E3_HEMS", ["type:hems"]),
-            (self.asElectricalEnergySystem, r"E3_TCU10_x07", ["type:tcu"]),
-            (self.asElectricalEnergySystem, r"E3_EEBus", ["type:eebus"]),
-            (self.asElectricalEnergySystem, r"E3_VitoCharge_03", ["type:energy_storage"]),
+            (self.asGateway, r"E3_TCU10_x07", ["type:gateway;TCU300"]),
```

Comment:
The TCU could probably also be ignored in the HA integration, right?

https://github.com/home-assistant/core/blob/dev/homeassistant/components/vicare/const.py#L18

Reply:
Yes, the TCU does not provide any information

Reply:
Do you create the PR in the Home Assistant Repo?